CONTRIBUTING.md

```diff
@@ -1,52 +1,68 @@
 # Contributing to OpenEBS
 
-Great!! We are always on the lookout for more OpenEBS hackers. You can get started by reading this [Overview](./contribute/design/README.md)
+Great!! We are always on the lookout for more OpenEBS hackers. You can get started by reading this [overview](./contribute/design/README.md)
 
-OpenEBS is innovation in OpenSource. You are welcome to contribute in any way you can and all the help you can provide is very much appreciated. 
+Firstly, if you are unsure or afraid of anything, just ask or submit the issue or pull request anyways. You won't be yelled at for giving your best effort. The worst that can happen is that you'll be politely asked to change something. We appreciate any sort of contributions and don't want a wall of rules to get in the way of that.
 
-- Raise Issues on either the functionality or documentation
-- Submit Changes to Improve Documentation 
-- Submit Proposals for new Features/Enhancements 
-- Submit Changes to the Source Code
+However, for those individuals who want a bit more guidance on the best way to contribute to the project, read on. This document will cover all the points we're looking for in your contributions, raising your chances of quickly merging or addressing your contributions.
+
+That said, OpenEBS is an innovation in open-source. You are welcome to contribute in any way you can and all the help provided is very much appreciated. 
+
+- Raise issues to request new functionality, fix documentation or for reporting bugs.
+- Submit changes to improve documentation. 
+- Submit proposals for new features/enhancements.
+- Submit fixes for bugs reported/found. 
+- Submit changes to the source code.
+- Solve existing issues.
 
 There are just a few simple guidelines that you need to follow before providing your hacks. 
 
 ## Raising Issues
 
-When Raising issues, please specify the following:
-- Setup details (like hyperconverged/dedicated), orchestration engine - kubernetes, docker swarm etc,. 
-- Scenario where the issue was seen to occur
-- If the issue is with storage, include maya version, maya osh-status and maya omm-status.
-- Errors and log messages that are thrown by the software
-
+When raising issues, please specify the following:
+- Setup details (like hyperconverged/dedicated), orchestration engine - Kubernetes, Docker Swarm etc. 
+- A Scenario where the issue occurred (with details on how to reproduce it)
+- If the issue is with storage, include maya version, maya osh-status, and maya omm-status.
```

Comment:
Remove this line please. osh/omm are old concepts..

Reply:
Line completely removed.